osx/src/main/java/ch/cyberduck/ui/cocoa/datasource/BrowserTableDataSource.java

```diff
@@ -418,19 +419,21 @@ public boolean acceptDrop(final NSTableView view, final Path destination, final
                     final Path renamed = new Path(destination, next.getName(), next.getType(), new PathAttributes(next.attributes()).withVersionId(null));
                     files.put(next, renamed);
                 }
-                if(pasteboard.getBookmark().compareTo(controller.getSession().getHost()) != 0) {
-                    // Drag to browser windows with different session or explicit copy requested by user.
-                    final Host target = controller.getSession().getHost();
-                    controller.transfer(new CopyTransfer(pasteboard.getBookmark(), target, files),
-                        new ArrayList<Path>(files.values()), false);
-                }
-                else if(info.draggingSourceOperationMask().intValue() == NSDraggingInfo.NSDragOperationCopy.intValue()) {
-                    // The file should be copied
-                    new CopyController(controller).copy(files);
+                if(new AbstractHostCollection.ProfilePredicate(pasteboard.getBookmark()).test(controller.getSession().getHost())) {
```

Comment:
Any need to include the default path in the test as `ProfilePredicate` does not contain it?

Reply:
The fix is to *not* include the default path as this is set when choosing _New Browser_ in the context menu.